README.md

```diff
@@ -30,10 +31,12 @@ Package contains several configuration presets:
  - Evonet - masternode with Evonet configuration
  - Testnet - masternode with testnet configuration
 
-There are two ways to apply a present:
+There are two ways to apply a preset:
  1. Rename corresponding dotenv file (i.e. `.env.local`) to `.env`
  2. Add `--env-file` option to `docker-compose` command
 
+Edit your chosen preset file to specify configuration variables before starting the masternode.
+
 ### Start
 
 In order to run a masternode use Docker Compose:
```

Comment:
Should we describe here how to start a full node or masternode for specific preset?

Also, I would describe how to register masternode.

Reply:
Added syntax and examples for individual commands, is this what you had in mind?

Reply:
Not really, please check the comments above.